tests/open_alchemy/helpers/test_peek/test_peek_extension.py

```diff
@@ -646,15 +810,23 @@ def test_mixins(schema, schemas, expected_mixins):
     assert mixins == expected_mixins
 
 
+@pytest.mark.parametrize(
+    "schema",
+    [
+        ({"x-dict-ignore": "True"}),
+        pytest.param(
+            {"x-open-alchemy-dict-ignore": "True"},
+            marks=pytest.mark.xfail,
```

Comment:
This also didn't fail for me when I ran it?

Reply:
Actually for me neither 🤔 

I removed the marker.